_docs_integrate/integration-example.md

```diff
@@ -162,8 +157,8 @@ Even though the Requests are validated during the RelationshipTemplate creation,
 
 ### Connector: Create a RelationshipTemplate
 
-If the Response is successful, we can create the RelationshipTemplate.
-To do so, we use the `content` we just validated.
+If the response successfully confirms all attributes, we can create the RelationshipTemplate.
```

Comment:
```suggestion
If the response is successful, we can create the RelationshipTemplate.
```

it sounds like this would only validate attributes, but it does way more things.

Reply:
how else could you write it. my problem is that the response comes back successfully (201), even if there are problems in it. I would like to make that clearer

Reply:
wtb `if the Connector states your Request as valid`?